docs/data-sources/organization.md

```diff
@@ -3,18 +3,18 @@
 page_title: "aiven_organization Data Source - terraform-provider-aiven"
 subcategory: ""
 description: |-
-  Retrieves information about an organization from Aiven.
+  Gets information about an organization.
 ---
 
 # aiven_organization (Data Source)
 
-Retrieves information about an organization from Aiven.
+Gets information about an organization.
 
 ## Example Usage
 
 ```terraform
-data "aiven_organization" "organization1" {
-  name = "<ORGANIZATION_NAME>"
+data "aiven_organization" "main" {
+  name = "Example organization"
```

Comment:
I think we use placeholders in other datasources, maybe a more standardized approach should be used, or this should be changed elsewhere too?

Reply:
You're right and I considered sticking with that, but after some research and consideration about what might be easier to work with and read I decided to go with this. I have an epic to go through all of the resources and data sources to standardize the descriptions, example usage, import syntax, etc. :)

Reply:
Ah, good. Thank you for the clarification!